src/pages/docs/chat/index.mdx

```diff
@@ -9,13 +9,72 @@ Ably Chat is a product dedicated to making it quick and easy to build chat funct
 
 The Chat SDK contains a set of purpose-built APIs that abstract away the complexities involved in how you would architect chat features. It utilizes Ably's platform to benefit from all of the same performance guarantees and scaling potential.
 
-* [Getting started: Chat in JavaScript / TypeScript](/docs/chat/getting-started/javascript)
-* [Getting started: Chat in React](/docs/chat/getting-started/react)
-* [Getting started: Chat UI Kits in React](/docs/chat/getting-started/react-ui-components)
-* [Getting started: Chat in React Native](/docs/chat/getting-started/react-native)
-* [Getting started: Chat in Android](/docs/chat/getting-started/android)
-* [Getting started: Chat in Kotlin (JVM)](/docs/chat/getting-started/jvm)
-* [Getting started: Chat in Swift](/docs/chat/getting-started/swift)
+## Why Chat
+
+### Built for Serious Scale & Reliability
+
+Ably Chat is powered by Ably's globally distributed platform that handles **550 billion messages per month** and **1 billion+ connections per month**, serving the world's most demanding workloads with **100% uptime over 5+ years**. With a global edge network featuring **635 points of presence**, Ably Chat delivers messages with **&lt;50ms median roundtrip latency** and provides automatic multi-region failover to ensure your chat experiences are always available.
+
+### Cost-Effective Scalability
+
+Scale without penalty using Ably's usage-based pricing model that charges by-the-minute for what you actually use. The platform includes built-in cost optimization features like **message batching** to reduce consumption, and **volume discounts** that lower your overall cost as you grow. This means you can power more chat functionality within budget without stifling innovation for fear of cost.
+
+### Developer Productivity
+
+Get building with intuitive APIs and comprehensive SDKs across multiple platforms. The Chat SDK abstracts away architectural complexity, enabling your team to rapidly ship exceptional chat experiences. Pre-built **React UI Kits** and **React Hooks** accelerate development even further, letting you focus on creating unique experiences rather than building chat infrastructure from scratch.
+
+### Enterprise-Ready
+
+Ably Chat supports enterprise requirements with **HIPAA and SOC2 compliance** options, dedicated clusters, custom domains (CNAME), SSO integration, and premium support with **sub-15 minute response times**. Whether you're building for thousands or millions of users, Ably Chat scales with your business needs.
+
+## Get started now <a id="get-started"/>
+
+<Tiles>
```

Comment:
Need some sort of run on, or intro text about SDK availability (similar to what you covered in the dev productivity section. 

We should likely still pull UI kits out separately too IMO.